socorro/schemas/crash_report.json

```diff
@@ -324,6 +324,76 @@
         },
+        "memory_measures": {
+            "type": "object",
+            "description": "Memory measurements collected or computed from the memory_report.",
+            "properties": {
+                "explicit": {
+                    "type": ["integer", "null"],
+                    "description": "The 'explicit' measurement from about:memory."
+                },
+                "gfx_textures": {
+                    "type": ["integer", "null"],
+                    "description": "The 'gfx-textures' measurement from about:memory."
+                },
+                "ghost_windows": {
+                    "type": ["integer", "null"],
+                    "description": "The 'ghost-windows' measurement from about:memory."
+                },
+                "heap_allocated": {
+                    "type": ["integer", "null"],
+                    "description": "The 'heap-allocated' measurement from about:memory."
+                },
+                "heap_overhead": {
+                    "type": ["integer", "null"],
+                    "description": "The 'heap-overhead' measurement from about:memory."
+                },
+                "heap_unclassified": {
+                    "type": ["integer", "null"],
+                    "description": "The 'heap-unclassified' measurement from about:memory."
+                },
+                "host_object_urls": {
+                    "type": ["integer", "null"],
+                    "description": "The 'host-object-urls' measurement from about:memory."
+                },
+                "images": {
+                    "type": ["integer", "null"],
+                    "description": "The 'images' measurement from about:memory."
+                },
+                "js_main_runtime": {
+                    "type": ["integer", "null"],
+                    "description": "The 'js-main-runtime' measurement from about:memory."
+                },
+                "private": {
+                    "type": ["integer", "null"],
+                    "description": "The 'private' measurement from about:memory."
+                },
+                "resident": {
+                    "type": ["integer", "null"],
+                    "description": "The 'resident' measurement from about:memory."
+                },
+                "resident_unique": {
+                    "type": ["integer", "null"],
+                    "description": "The 'resident-unique' measurement from about:memory."
+                },
+                "system_heap_allocated": {
+                    "type": ["integer", "null"],
+                    "description": "The 'system-heap-allocated' measurement from about:memory."
+                },
+                "top_non_detached": {
+                    "type": ["integer", "null"],
+                    "description": "The 'top-non-detached' measurement from about:memory."
```

Comment:
Ditto.

Also, please say 'top(none)/detached' instead of 'top-none-detached', because that's how it shows up in about:memory. (I'm going to do likewise when displaying it in report_index.html.)